doc/pylint.rst

```diff
@@ -1,36 +1,97 @@
-Cancel analyzing a file with the :guilabel:`Stop` button, and if analysis fails, click the :guilabel:`Output` button to find out why.
+=======================
+Using the code analyzer
+=======================
 
-To go directly to the file and line in the :doc:`editor` highlighted by a failed check, just click its name.
-You can click the dropdown or press the :kbd:`Down Arrow` key in the filename field to view results of previous analyses; the number of recent runs Spyder should remember can be customized in the :guilabel:`History` dialog from the :guilabel:`Static Code Analysis` context menu.
+You can select the desired file to analyze directly in the :doc:`editor` by clicking anywhere within it.
+To run it, click the :guilabel:`Analyze` button in the pane, press the configurable code analysis shortcut (:kbd:`F8` by default) or select :menuselection:`Source --> Run code analysis` from the menu bar.
 All standard checks are run by default.
-You can turn certain messages off at the line, block or file/module level by adding a ``# pylint: disable=<list of message names>`` comment at the respective level, where the ``<...>`` portion should be replaced with a comma-separated list (or single value) without the ``<``/``>`` of Pylint message names (*e.g.* ``multiple-statements``, or ``fixme, line-too-long``; you can also use error codes like ``C0321`` although names are preferred for clarity).
+To go directly to line in the :doc:`editor` highlighted by a failed check, just click its name.
+
+.. image:: images/pylint/code_analysis_editor.gif
+   :alt: Spyder Pylint pane, showing running analysis and clicking failed check
+
+|
+
+You can manually enter the Python module or package path you'd like it to check in the path entry textbox in the :guilabel:`Code Analysis` pane. 
+The analyzer works with both ``.py`` (or ``.pyw``) Python scripts and whole Python packages (directories containing an :file:`__init__.py` file).
+
+.. image:: images/pylint/code_analysis_file.gif
+   :alt: Spyder Pylint pane, showing running analysis browsing file
+
+|
+
+Cancel analyzing a file with the :guilabel:`Stop` button, and if analysis fails, click the :guilabel:`Output` button to find out why. 
+
+.. image:: images/pylint/code_analysis_stop.gif
+   :alt: Spyder Pylint pane, showing running analysis browsing file
+
+|
+
+You can click the dropdown or press the :kbd:`Down Arrow` key in the filename field to view results of previous analyses; 
+
+
+
+===================
+Contex menu options
+===================
+The number of recent runs Spyder should remember can be customized in the :guilabel:`History` dialog from the :guilabel:`Code Analysis` context menu.
+
+.. image:: images/pylint/code_analysis_history.gif
+   :alt: Spyder Pylint pane, customizing history dialog
+
+|
+
+You can also expand or collapse one or all the sections in the pane by using the corresopnding options in the context menu.
+
+.. image:: images/pylint/code_analysis_expand_collapse.gif
```

Comment:
The text and section title says to use the options in the context menu, but you actually show using the pane options menu instead. I would suggest changing the text and section title accordingly since these options make more sense from the options menu rather than being directly contextual, but idk. Otherwise, and being sped up by maybe 1.5x or so, it looks good.

Reply:
> The text and section title says to use the options in the context menu, but you actually show using the pane options menu instead

I agree and left the corresponding suggestions above.

> and being sped up by maybe 1.5x or so

Perhaps 1.25x is enough?